java-frontend/src/main/java/org/sonar/java/bytecode/se/BytecodeEGWalker.java

```diff
@@ -867,7 +870,8 @@ private void executeCheckEndOfExecution() {
     if(!isStaticMethod) {
       // Add a sv for "this"
       SymbolicValue thisSV = constraintManager.createSymbolicValue((Instruction) null);
-      state = currentState.addConstraint(thisSV, ObjectConstraint.NOT_NULL).put(0, thisSV);
+      Type classType = semanticModel == null ? Symbols.unknownType : semanticModel.getClassType(signature.substring(0, signature.indexOf('#')));
```

Comment:
`semanticModel` should never be null in production , and tests should be adapted to have it instead